sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -247,6 +247,10 @@ void gnrc_ndp_nbr_adv_handle(kernel_pid_t iface, gnrc_pktsnip_t *pkt,
 
         opt_offset += (opt->len * 8);
         sicmpv6_size -= (opt->len * 8);
+
+        if (sicmpv6_size < 0) {
+            DEBUG("ndp: Option parsing out of sync.\n");
+        }
```

Comment:
If you really want to add this message, surround it with `#if ENABLE_DEBUG` please. Also why not the other message handle functions then, too?